int32 id
int64 x_pos
int64 y_pos
int64 z_pos
int64[4] anchor_distances
